frontend/src/app/shared/functions/get-image-aspect-ratio.ts

```diff
@@ -1,3 +1,7 @@
+/**
+ * @returns wether the a Promise that resolves with the aspect ratio of the image
+ * it rejects if the image Url is invalid
+ */
```

Comment:
I don't know why, but this doesn't work. It rejects for URLs that exist. You can test this with any remote URL, e.g. `https://hpi.de/typo3conf/ext/hpi_distribution/Resources/Public/Default/img/hpi_logo_srgb_wb_sl1_web80.png` (a low-res HPI logo). It exists but isn't recognized. The error is `Failed to fetch`.